kafka-handler/src/test/org/apache/hadoop/hive/kafka/KafkaRecordIteratorTest.java

```diff
@@ -304,7 +305,7 @@ private static void sendData(List<ConsumerRecord<byte[], byte[]>> recordList, @N
   @After public void tearDown() {
     this.kafkaRecordIterator = null;
     if (this.consumer != null) {
-      this.consumer.close();
```

Comment:
Why are we explicitly passing a zero duration? Can't we use the existing method with the default delay?